cyclonedx-lib/src/temurin/sbom/TemurinGenSBOM.java

```diff
@@ -111,6 +113,11 @@ public static void main(final String[] args) {
                 writeJSONfile(bom, fileName);
                 break;
 
+            case "signSBOM":                              // Signs JSON file
+                bom = signBom();
+                writeJSONfile(bom, fileName);
```

Comment:
does not think need to call `writeJSONfile` if it is to sign file. 

instead something like `new JSONObjectWriter().setSignature(new JSONAsymKeySigner(privateKey, publicKey, null));`